openmeetings-service/src/main/java/org/apache/openmeetings/service/calendar/caldav/handler/WebDAVSyncHandler.java

```diff
@@ -0,0 +1,149 @@
+    }
+
+    public OmCalendar updateItems(){
+        boolean additionalSyncNeeded = false;
+
+        SyncMethod syncMethod = null;
+
+        try {
+            DavPropertyNameSet properties = new DavPropertyNameSet();
+            properties.add(DavPropertyName.GETETAG);
+
+            //Create report to get
+            SyncReportInfo reportInfo = new SyncReportInfo(calendar.getToken(), properties,
+                    SyncReportInfo.SYNC_LEVEL_1);
+            syncMethod = new SyncMethod(path, reportInfo);
+            client.executeMethod(syncMethod);
+
+            if(syncMethod.succeeded()){
+
+                List<String> currenthrefs = new ArrayList<>();
+
+                //Map of Href and the Appointments, belonging to it.
+                Map<String, Appointment> map = listToMap(appointmentDao.getAppointmentHrefsinCalendar(calendar.getId()),
+                        appointmentDao.getAppointmentsinCalendar(calendar.getId()));
+
+                for(MultiStatusResponse response: syncMethod.getResponseBodyAsMultiStatus().getResponses()){
+                    int status = response.getStatus()[0].getStatusCode();
+                    if(status == DavServletResponse.SC_OK){
+                        Appointment a = map.get(response.getHref());
+
+                        //Old Event to get
+                        if(a != null){
+                            String origetag = a.getEtag(),
+                                    currentetag = CalendarDataProperty.getEtagfromResponse(response);
+
+                            //If event modified, only then get it.
+                            if(!currentetag.equals(origetag))
+                                currenthrefs.add(response.getHref());
+                        }
+                        //New Event, to get
+                        else
+                            currenthrefs.add(response.getHref());
+                    }
+                    else if(status == DavServletResponse.SC_NOT_FOUND){
+                        //Delete the Appointments not found on the server.
+                        Appointment a = map.get(response.getHref());
+
+                        //Only if the event exists on the database, delete it.
+                        if(a != null) {
+                            appointmentDao.delete(a, calendar.getOwner().getId());
+                        }
+                    }
+                    else if (status == DavServletResponse.SC_INSUFFICIENT_SPACE_ON_RESOURCE){
+                        additionalSyncNeeded = true;
+                    }
+                }
+
+
+                MultigetHandler multigetHandler = new MultigetHandler(currenthrefs, path,
+                        calendar, client, appointmentDao, utils);
+                multigetHandler.updateItems();
+
+                //Set the new token
+                calendar.setToken(syncMethod.getResponseSynctoken());
+            } else if (syncMethod.getStatusCode() == DavServletResponse.SC_FORBIDDEN ||
```

Comment:
`import static DavServletResponse.SC_FORBIDDEN` will shorten the code and will make it more readable